recorder:
  audio_data: ["harmoni_microphone_default"]
  video_data: ["harmoni_camera_default"]
  audio_video_data: ["harmoni_microphone_1", "harmoni_camera_1"] #It is an array of 2 items: array[0] refers to audio data input, array[1] refers to video data input
  outdir: "/root/harmoni_catkin_ws/src/HARMONI/harmoni_core/harmoni_recorder/temp_data/" #folder where audio and video data are stored

